packages/wonder-blocks-dropdown/src/components/multi-select.tsx

```diff
@@ -607,6 +631,10 @@ const MultiSelect = (props: Props) => {
     const filteredItems = getMenuItems(allChildren);
     const isDisabled = numEnabledOptions === 0 || disabled;
 
+    if (open && isFilterable) {
+        handleAnnouncement(labels.someSelected(filteredItems.length));
```

Comment:
Would it make sense to trigger this announcement when an event occurs or when the data changes?

Reply:
I looked into that too...this code was the biggest culprit in triggering infinite loops as it's in the main component thread. The filtered items aren't tracked with state, so I'd have to find a safe way to do that. The change event for this doesn't do any filtering on children. I could try moving it back into dropdown-core, but I hoped to keep all announcement handling centralized in `SingleSelect` and `MultiSelect`, respectively. I'm open to suggestions as this was really tricky!

Reply:
Ahh I see, this is what you were mentioning the other day! I'll poke around and get back to you 😄

Reply:
Tinkered around a bit in SingleSelect and was able to reduce the announcement to only when the selectedValue/children changes and when the filtered items length changes! https://github.com/Khan/wonder-blocks/compare/combobox-announcer-v2...combobox-tinkering?expand=1 

(Note: I duplicated some logic to figure out the `selectedOption` within the `useEffect` to avoid additional renders. We could memoize that logic since other things use that logic too!)

Reply:
Thanks so much for this Bea!! I appreciate you taking the time to tinker with it. I'm testing it out in Safari and VO and it is working so far... I'll keep playing with it and a MultiSelect version.